ui/apps/platform/src/Containers/Integrations/IntegrationForm/IntegrationForm.tsx

```diff
@@ -37,6 +37,7 @@ import ClusterInitBundleIntegrationForm from './Forms/ClusterInitBundleIntegrati
 import SignatureIntegrationForm from './Forms/SignatureIntegrationForm';
 
 import './IntegrationForm.css';
+import { isUserResource } from '../../AccessControl/traits';
```

Comment:
Suggestion that affects some other `import` statements.

Instead of relative `..` chain to files outside of a container, we usually make it obvious:

```ts
import { isUserResource } from 'Containers/AccessControl/traits';
```

Linter will make you move the `import` above any relative imports.
```